File: examples/modulith-clinical-tool-jpa/src/main/java/io/zenwave360/example/clinicaltool/modules/clinical/core/domain/events/PatientCreated.java
package io.zenwave360.example.clinicaltool.modules.clinical.core.domain.events;

import io.zenwave360.example.clinicaltool.modules.clinical.core.domain.*;
import jakarta.validation.constraints.*;
import java.io.Serializable;
import java.math.*;
import java.time.*;
import java.util.*;

/**
 *
 */
@lombok.Getter
@lombok.Setter
@org.springframework.modulith.NamedInterface("events.PatientCreated")
public class PatientCreated implements Serializable {

    @java.io.Serial
    private static final long serialVersionUID = 1L;

    private Patient patient;
}

File: examples/modulith-klinical-tool-jpa/src/main/kotlin/io/zenwave360/example/clinicaltool/modules/clinical/core/domain/events/PatientCreated.kt
package io.zenwave360.example.clinicaltool.modules.clinical.core.domain.events

import io.zenwave360.example.clinicaltool.modules.clinical.core.domain.*
import jakarta.validation.constraints.*
import java.io.Serializable
import java.math.*
import java.time.*
import java.util.*

/**  */
@org.springframework.modulith.NamedInterface("events.PatientCreated")
data class PatientCreated(var patient: Patient? = null) : Serializable {

    companion object {
        private const val serialVersionUID: Long = 1L
    }
}
